cmd/kube-proxy/app/server.go

```diff
@@ -51,24 +51,26 @@ import (
 
 // ProxyServerConfig contains configures and runs a Kubernetes proxy server
 type ProxyServerConfig struct {
-	BindAddress        net.IP
-	HealthzPort        int
-	HealthzBindAddress net.IP
-	OOMScoreAdj        int
-	ResourceContainer  string
-	Master             string
-	Kubeconfig         string
-	PortRange          util.PortRange
-	HostnameOverride   string
-	ProxyMode          string
-	IptablesSyncPeriod time.Duration
-	ConfigSyncPeriod   time.Duration
-	NodeRef            *api.ObjectReference // Reference to this node.
-	MasqueradeAll      bool
-	CleanupAndExit     bool
-	KubeAPIQPS         float32
-	KubeAPIBurst       int
-	UDPIdleTimeout     time.Duration
+	BindAddress                    net.IP
+	HealthzPort                    int
+	HealthzBindAddress             net.IP
+	OOMScoreAdj                    int
+	ResourceContainer              string
+	Master                         string
+	Kubeconfig                     string
+	PortRange                      util.PortRange
+	HostnameOverride               string
+	ProxyMode                      string
+	IptablesSyncPeriod             time.Duration
+	ConfigSyncPeriod               time.Duration
+	NodeRef                        *api.ObjectReference // Reference to this node.
+	MasqueradeAll                  bool
+	CleanupAndExit                 bool
+	KubeAPIQPS                     float32
+	KubeAPIBurst                   int
+	UDPIdleTimeout                 time.Duration
+	ConntrackMax                   int
```

Comment:
As previous - it can't be any finer than seconds, so I thought simpler was better.  If you think it's clearer I can change it, but it will involve getting the value back out by .Seconds() (float64) and then casting to int.